### PR TITLE
Add global option and fix bug in line 69 of bash script

### PR DESCRIPTION
**Pull request by eurunuela**
- This PR adds the option to read the labels of all Brainhack Global event projects.
- It also fixes a little bug that made reading labels fail.

### CODE REVIEW

File: tools/pull_issues.sh [hunk 1]
```diff
@@ -75,7 +81,7 @@ done
 cd ..
 
 # Check if filename has a slash (path). Move the file one directory up if it's just a filename.
-if [[ "${OUTFILE}" != *\/*  ]] || [[ "${OUTFILE}" != *\\* ]]
+if [[ "${OUTFILE}" != *\/*  ]] || [[ "${OUTFILE}" != *\\* ]]
```
Comment (jhlegarreta) [inline tools/pull_issues.sh:84]: What is changed on this line?

**Review by jhlegarreta — COMMENTED** [review]
Have not tested, but looks good to me.

Can the in-line comment be answered? Thanks.

File: tools/pull_issues.sh (hunk 1 above)
Comment (eurunuela) [reply tools/pull_issues.sh:84]: There was a missing space that raised an error.

**Comment by eurunuela** [comment]
> Have not tested, but looks good to me. Can a corresponding test case, even if manually added, be added and checked, please?

It works on my end.

> Can the in-line comment be answered? Thanks.

Done.

**Comment by jhlegarreta** [comment]
> It works on my end.

I trust, but please add a test case to the data so that it can be tested.

**Comment by jhlegarreta** [comment]
>> Have not tested, but looks good to me. Can a corresponding test case, even if manually added, be added and checked, please?

> It works on my end.

I have checked out the branch, called the script with the existing participant data and project data and the Python script fails [here­](https://github.com/brainhackorg/brainmatch/blob/main/brainmatch/brainmatch.py#L239).

**Comment by eurunuela** [comment]
Hi @jhlegarreta , thanks for raising that point. I meant the shell script worked on my end, having completely forgotten that the python script needed to be changed as well. I'll commit some changes now.

**Comment by eurunuela** [comment]
I don't quite understand what you mean by a test case, but the line I added works. It basically considers all `bhg:` labels for the scores. In the end, the `bhg:` prefix is only used for labeling the events afaik.

**Comment by jhlegarreta** [comment]
> I don't quite understand what you mean by a test case, but the line I added works. It basically considers all bhg: labels for the scores. In the end, the bhg: prefix is only used for labeling the events afaik.

Basically, testing that what you expect as the output is what the tool will output if the changes are merged. In this case, after some time, I realized that `bhg:global` is not expected to exist as a label in the BHG 2020 repository (but correct me if I'm wrong). In the former case, by testing I mean testing the Python script using the `bhg:global` option, and updating or possibly adding new expected values (files, in case `bhg:global` is effectively not a label in the BHG 2020 repository) for that case to the existing [test data](https://github.com/brainhackorg/brainmatch/tree/main/data) so that we know what the expected values are, and are able to detect undesired deviations in future changes.

**Comment by eurunuela** [comment]
I've edited the `projects.tsv` file we use for testing and added the results of using `bhg:global` as a parameter: `expected_match_global.csv` and `expected_match_global_top.csv`.

**Comment by jhlegarreta** [comment]
Checked locally: working well. Thanks :100:. Merging.

**Pull request merged**